cardano-cli/src/Cardano/CLI/Shelley/Run/Address/Info.hs

```diff
@@ -57,23 +60,30 @@ runAddressInfo addrTxt mOutputFp = do
               , aiEra = "byron"
               , aiEncoding = "base58"
               , aiAddress = addrTxt
+              , aiBase16 = asBase16 payaddr
               }
           ShelleyAddress{} ->
             pure $ AddressInfo
               { aiType = "payment"
               , aiEra = "shelley"
               , aiEncoding = "bech32"
               , aiAddress = addrTxt
+              , aiBase16 = asBase16 payaddr
               }
 
-      Just (Right _stakeaddr) ->
+      Just (Right addr) ->
         pure $ AddressInfo
           { aiType = "stake"
           , aiEra = "shelley"
           , aiEncoding = "bech32"
           , aiAddress = addrTxt
+          , aiBase16 = asBase16 addr
           }
 
     case mOutputFp of
       Just (OutputFile fpath) -> liftIO $ LBS.writeFile fpath $ encodePretty addrInfo
       Nothing -> liftIO $ LBS.putStrLn $ encodePretty addrInfo
+
+  where
+    asBase16 :: SerialiseAsRawBytes a => a -> Text
+    asBase16 = Text.decodeUtf8 . Base16.encode . serialiseToRawBytes
```

Comment:
`Base16.encode . serialiseToRawBytes` can be replaced with `serialiseToRawBytesHex` in `Cardano.Api.Typed` which will allow you to remove the Base16 import

Reply:
@Jimbo4350 done.